### DISCUSSION TITLE
Leveraging Full Screen for Tabular Data

### DISCUSSION THREAD

**Discussion by nara-golfdistrict-com** [juspay/hyperswitch#3636] [Ideas / Feature Requests]
There are several screens in the HyperSwitch portal where the full screen width is not utilized. It will be beneficial if we can leverage the full screen width, so that we can see more data. See below screenshot as an example.
![image](https://github.com/juspay/hyperswitch/assets/145275696/29ee8b9b-b205-4b11-b9c0-954c0f336511)

**Comment by SanchithHegde** [comment]
This is related to the Hyperswitch Control Center.

CC: @juspay/hyperswitch-control-center

**Comment by dvenga** [comment]
@nara-golfdistrict-com - Utilising the full screen was creating issues for different screen sizes - including mobile, even after making the screen responsive. This was the reason why we decided to have a fixed width - which will work in most of the screen sizes, without breaking the experience.
